aws/aws.go

```diff
@@ -177,8 +191,21 @@ func NukeAllResources(account *AwsAccountResources, regions []string) error {
 
 		resourcesInRegion := account.Resources[region]
 		for _, resources := range resourcesInRegion.Resources {
-			if err := resources.Nuke(session); err != nil {
-				return errors.WithStackTrace(err)
+			if len(resources.ResourceIdentifiers()) >= 500 {
+				logging.Logger.Infoln("Terminating Resources in batches")
```

Comment:
"There are more than XXX resources to terminate (YYY total), so terminating resources in batches"